frontend/src/scenes/scenes.ts

```diff
@@ -271,6 +271,10 @@ export const sceneConfigurations: Record<Scene, SceneConfig> = {
         projectBased: true,
         layout: 'plain',
     },
+    [Scene.OnboardingProductIntroduction]: {
+        projectBased: true,
+        name: 'Product onboarding',
```

Comment:
```suggestion
        name: 'Product onboarding intro',
```
??

Reply:
This shows up in the UI so I just liked "Product onboarding"